src/vs/code/electron-main/window.ts

```diff
@@ -686,7 +686,7 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 
 		// Load URL
 		perf.mark('main:loadWindow');
-		this._win.loadURL(this.getUrl(configuration));
+		this._win.loadURL('vscode-file://localhost' + this.getUrl(configuration).slice(7));
```

Comment:
@alexdima @deepak1556 same, I would expect `getUrl` to return the proper URL without slicing and concat.